compose/compose-file.md

```diff
@@ -1594,7 +1443,7 @@ several options have been removed:
     `deploy`. Note that `deploy` configuration only takes effect when using
     `docker stack deploy`, and is ignored by `docker-compose`.
 
-#### Version 1 to 2.x
+### Version 1 to 2.x
```

Comment:
@dnephin I've just moved everything like that into a separate versioning/upgrade doc which I'm linking to from all the other Compose reference files. I have to fix some things in the left side menu but I'll push this all in about an hour or two, then take a look and see what you think. I'll ping you. I think it will be cleaner, and simpler.